### ISSUE TITLE
 Color contrast in Submit button on New item form  is not sufficient

### ISSUE THREAD

**Issue by reetagithub** [DSpace/dspace-angular#2912]
To reproduce this, you can either WAVE the page (a static accessibility testing tool, an addon to the browser) of insert the hexadecimal values of the colors to WebAIM's [ContrastChecker](https://webaim.org/resources/contrastchecker/). 

**Related work**
#2715 handles sccessibility enhancenents of the UI

**Comment by tdonohue** [comment]
I believe this is related to the "+ Deposit" button on the Submission form.

While I cannot seem to get WAVE to throw a contrast error on that button (at least it doesn't work with my Chrome plugin), I have been able to verify that the contrast is failing via https://webaim.org/resources/contrastchecker/

The background of that button is #92c642 while the foreground text is white #FFFFFF.  This fails the contrast checker above.
On hover over, the background changes to #7ead34 and the foreground text remains white.  This also seems to fail.

One possible quick fix might be to change the text to **black** (#000000)

Needs a volunteer.

**Comment by pcg-kk** [comment]
Hi, the color for the `.btn.btn-success` is different.

Currently, it is:
![image](https://github.com/DSpace/dspace-angular/assets/165835247/f8fcc321-1ec9-47a3-90da-8e3b7aa4e5ae)

As the `$success` is `#92c642` but darken means: `#5a7c26`

This means that the contrast is over `4.5:1`:
![image](https://github.com/DSpace/dspace-angular/assets/165835247/1c30db4b-56c6-432b-9eef-f6e8f45cdd24)

**Comment by tdonohue** [comment]
@pcg-kk : You are correct.  Somehow I was looking at the wrong style/color here.

On the submission form, the "+ Deposit" button has a #5a7c26 background with white text.  That passes color contrast.  When you hover over it, the background gets even darker (#3e551a).

So, this appears to already be solved.   Closing, as this is no longer a bug.
